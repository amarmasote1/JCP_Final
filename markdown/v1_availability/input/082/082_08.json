{
    "reservation": [
        {
            "storeID": "0951",
            "itemID": "58078781025",
            "uom": "EACH",
            "shippable": "true",
            "procurementAllowed": "false",
            "qty": "3",
            "lineID": "1",
            "reservationID": "R-v1-R002"
        }
    ]
}