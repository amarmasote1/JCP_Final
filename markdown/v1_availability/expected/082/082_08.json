{"status":200,"message":"Success","body":[{"shortageQty":0,"reservedQty":3,"partiallyReservedQty":0,"problemType":"SUCCESS","orgCode":"JCP","requestLine":{"lineID":"1","itemID":"5807878-1025","uom":"EACH","bopus":false,"shippable":true,"deliveryMethod":null,"increaseCapacity":null,"shutdown":false,"qty":3,"storeID":"0951","channel":null,"reservationID":"R-v1-R002","procurementAllowed":false}}]}